### PR TITLE
Sdks 3086 privacy manifests

### PR DESCRIPTION
**Pull request by vahancouver**
[SDKS-3086](https://bugster.forgerock.org/jira/browse/SDKS-3086)

[iOS] Add Privacy Manifests to SDK

1. Update Google SDK to 7.1.0
2. Remove storage(diskSpace) from HardwareCollector
3. Add Privacy Manifests to all modules
4. Update Signals SDK to 5.2.3
5. Update FRFacebookSignIn Privacy Manifest to reflect Facebook iOS SDK privacy manifest
6. Change SPM dependency versions from exactVersion to upToNextMinorVersion

### CODE REVIEW

File: FRAuth/FRAuth/Device/HardwareCollector.swift [hunk 1]
```diff
@@ -31,7 +31,6 @@ public class HardwareCollector: DeviceCollector {
 //        result["activeCPU"] = pi.activeProcessorCount
 //        result["multitaskSupport"] = UIDevice.current.isMultitaskingSupported
         result["manufacturer"] = "Apple"
-        result["storage"] = Int(self.getDiskSpaceSize())
```
Comment (rodrigoareis) [inline FRAuth/FRAuth/Device/HardwareCollector.swift:34]: Update copyright

File: FRAuth/FRAuth/Device/HardwareCollector.swift [hunk 2]
```diff
@@ -66,24 +65,6 @@ public class HardwareCollector: DeviceCollector {
     }
     
     
-    /// Calculates current device's total disk size in MB
-    ///
-    /// - Returns: Current device's disk size in MB
-    func getDiskSpaceSize() -> Double {
```
Comment (rodrigoareis) [inline FRAuth/FRAuth/Device/HardwareCollector.swift:72]: Is this something used by any feature? Device Profile collector? Should we update any doc?

File: FRAuth/FRAuthTests/FRAuthSwiftTests/FRAuth/Device/FRDeviceCollectorTests.swift [hunk 3]
```diff
@@ -81,7 +81,7 @@ class FRDeviceCollectorTests: FRAuthBaseTest {
         XCTAssertNotNil((hardware["cpu"] as? Int))
 //        XCTAssertNotNil((hardware["activeCPU"] as? Int))
         XCTAssertNotNil((hardware["memory"] as? Int))
-        XCTAssertNotNil((hardware["storage"] as? Int))
+//        XCTAssertNotNil((hardware["storage"] as? Int))
```
Comment (rodrigoareis) [inline FRAuth/FRAuthTests/FRAuthSwiftTests/FRAuth/Device/FRDeviceCollectorTests.swift:84]: Update copyright

Comment (rodrigoareis) [reply FRAuth/FRAuthTests/FRAuthSwiftTests/FRAuth/Device/FRDeviceCollectorTests.swift:84]: Also, would nice to have a short comment why it has been commented out

File: FRAuth/FRAuthTests/FRAuthSwiftTests/FRAuth/Network/NetworkReachabilityMonitorTests.swift [hunk 4]
```diff
@@ -169,10 +169,14 @@ class NetworkReachabilityMonitorTests: FRAuthBaseTest {
         waitForExpectations(timeout: 60, handler: nil)
         
         // Then
+      if #available(iOS 13.0, *) {
```
Comment (rodrigoareis) [inline FRAuth/FRAuthTests/FRAuthSwiftTests/FRAuth/Network/NetworkReachabilityMonitorTests.swift:172]: Update copyright

**Review by rodrigoareis — CHANGES_REQUESTED** [review]
Overall changes looks good. Minor comments to address

File: FRAuth/FRAuth/Device/HardwareCollector.swift (hunk 1 above)
Comment (vahancouver) [reply FRAuth/FRAuth/Device/HardwareCollector.swift:34]: Fixed

File: FRAuth/FRAuthTests/FRAuthSwiftTests/FRAuth/Device/FRDeviceCollectorTests.swift (hunk 3 above)
Comment (vahancouver) [reply FRAuth/FRAuthTests/FRAuthSwiftTests/FRAuth/Device/FRDeviceCollectorTests.swift]: Fixed

File: FRAuth/FRAuthTests/FRAuthSwiftTests/FRAuth/Network/NetworkReachabilityMonitorTests.swift (hunk 4 above)
Comment (vahancouver) [reply FRAuth/FRAuthTests/FRAuthSwiftTests/FRAuth/Network/NetworkReachabilityMonitorTests.swift:172]: Fixed

**Review by spetrov — APPROVED** [review]

File: FRAuth/FRAuth/Device/HardwareCollector.swift (hunk 2 above)
Comment (rodrigoareis) [reply FRAuth/FRAuth/Device/HardwareCollector.swift:72]: no, that's ok

**Review by rodrigoareis — APPROVED** [review]
Changes looks good to me

**Pull request merged**